sdk/template/ci.yml

```diff
@@ -0,0 +1,40 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+      - sdk/core/
+      - eng/
+      - build.gradle
```

Comment:
Lets add the template as a trigger path.

Reply:
changed from sdk/core to sdk/template